src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -134,6 +134,15 @@ public void setInferConst(boolean value) {
   boolean inferTypes;
 
   private boolean useNewTypeInference;
+  
+  /**
+   * Relevant only when {@link #useNewTypeInference} is true, where we normally disable OTI errors.
+   * If you want both NTI and OTI errors in this case, set to true.
+   * E.g. if using using a warnings guard to filter NTI or OTI warnings in new or legacy code,
+   * respectively.
+   * This will be removed when NTI becomes entirely replaces OTI.
+   */
+  public boolean reportOtiErrorsUnderNti = false;
```

Comment:
Can you rename this field to reportOTIErrorsUnderNTI? Also, can you make it package private, and add a public setter method?

Add a comment on the setter method "Not dead code; used by the open-source users of the compiler.". If any internal dead-code-detection tool picks this up as dead code, I'd like to remember to not delete it accidentally.